internal/controller/httproute_controller.go

```diff
@@ -101,6 +102,10 @@ func (r *HTTPRouteReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&v1alpha1.GatewayProxy{},
 			handler.EnqueueRequestsFromMapFunc(r.listHTTPRoutesForGatewayProxy),
 		).
+		Watches(&v1beta1.ReferenceGrant{},
```

Comment:
Do we allow not installing RefereGrant?
 If not installed, an error will occur here.

Reply:
Yes, it is allowed to not install ReferenceGrant and an error will occur here if not installed ReferenceGrant.
Let me fix it.